source/game.gameconstants.bmx

```diff
@@ -1416,6 +1416,43 @@ End Type
 
 
 
+
+
+Type TVTProductionStep {_exposeToLua}
+	Const NOT_STARTED:Int = 0
+	Const PREPRODUCTION:int = 1
+	Const PREPRODUCTION_DONE:int = 2
+	Const SHOOTING:int = 3
+	Const SHOOTING_DONE:int = 4
+	Const FINISHED:Int = 5
+	Const ABORTED:Int = 6
```

Comment:
Interessehalber. In welchen Fällen wird eine Produktion abgebrochen?

Reply:
Bei der Pleite eines Spielers koennte das passieren.

Im Gegensatz zu "finished" gaebe es dann nicht immer XP.